projects/portal/src/app/views/accounts/account/account.component.html

```diff
@@ -1,34 +1,29 @@
 <h2 class="mb-0">Account Info</h2>
 <ng-container *ngIf="baseAccount === undefined || null; then empty; else exist"></ng-container>
-
 <ng-template #empty>
   <p class="ml-4">*This account has no denoms</p>
 </ng-template>
-
 <ng-template #exist>
   <mat-card class="mb-4" *ngIf="baseAccount">
     <mat-list>
       <mat-list-item>
         <span class="whitespace-nowrap">Sequence:</span>
         <span class="flex-auto"></span>
         <span class="ml-2 break-all">{{ baseAccount?.sequence }}</span>
+        <mat-divider [inset]="true"></mat-divider>
```

Comment:
What's this change?
The original looks more beautiful.

Reply:
Thank you for the advise.

Reply:
Thank you for the advise.